### DISCUSSION TITLE
Is mongoose 5.8 with mongoDB v5.x  Compatibility?

### DISCUSSION THREAD

**Discussion by ehgks0000** [Automattic/mongoose#14389] [Q&A] [answered]
I'm using mongoose 5.8 and mongoDB 4.4.
mongoDB version to 5.x, is it okay?

**Comment by vkarpov15** [comment, marked as answer]
Technically no, you would need to upgrade to Mongoose 5.10 to get full  MongoDB 4.4 support. [Here's Mongoose's official MongoDB server version compatibility table](https://mongoosejs.com/docs/compatibility.html). Upgrading to Mongoose 5.10 from 5.8 should be relatively straightforward, no known breaking changes.

**Reply by ehgks0000** [reply]
thx :)
